data/languages.yaml

```diff
@@ -557,6 +557,7 @@ pt:
         - hora
         - horas
     minute:
+        - min
```

Comment:
Let's try to keep full singular form as the start of the list and add other forms after.